bin/data-get.js

```diff
@@ -37,8 +37,9 @@ const run = async () => {
   try {
     const start = new Date()
     let savedPath
+    const itIsDatahubOrGitHub = parseDatasetIdentifier(identifier)
```

Comment:
As your variable name is `itIsDatahubOrGitHub`, it should return boolean so I would change this line to:

```
const parsedIdentifier = parseDatasetIdentifier(identifier)
```